Documentation/AdvancedBuildInstructions.md

```diff
@@ -100,6 +100,38 @@ By default, ccache will include the plugins themselves in file hashes. So if a p
 will change, and you will be stuck with an uncached build. This setting will prevent ccache from using plugins in the
 file hashes.
 
+## Building the flatpak
```

Comment:
Should Flatpak be used instead, for the occurrences in text? To bring it in line with how Ladybird and CMake are capitalized according to the project naming.